config.go

```diff
@@ -3,37 +3,39 @@ package quotacontrol
 import (
 	"time"
 
+	"github.com/0xsequence/quotacontrol/internal/toml"
 	"github.com/0xsequence/quotacontrol/middleware"
 	"github.com/0xsequence/quotacontrol/proto"
-	"github.com/goware/cachestore/redis"
 )
 
-type Duration struct {
-	time.Duration
-}
-
-func (d *Duration) UnmarshalText(text []byte) error {
-	var err error
-	d.Duration, err = time.ParseDuration(string(text))
-	return err
-}
-
 type Config struct {
-	Enabled       bool                `toml:"enabled"`
-	URL           string              `toml:"url"`
-	AuthToken     string              `toml:"auth_token"`
-	UpdateFreq    Duration            `toml:"update_freq"`
-	RateLimiter   middleware.RLConfig `toml:"rate_limiter"`
-	Redis         redis.Config        `toml:"redis"`
-	DefaultUsage  *int64              `toml:"default_usage"`
-	LRUSize       int                 `toml:"lru_size"`
-	LRUExpiration Duration            `toml:"lru_expiration"`
-	ErrorConfig   ErrorConfig         `toml:"error_config"`
+	Enabled       bool            `toml:"enabled"`
+	URL           string          `toml:"url"`
+	AuthToken     string          `toml:"auth_token"`
+	UpdateFreq    toml.Duration   `toml:"update_freq"`
+	RateLimiter   RateLimitConfig `toml:"rate_limiter"`
+	Redis         RedisConfig     `toml:"redis"`
+	DefaultUsage  *int64          `toml:"default_usage"`
+	LRUSize       int             `toml:"lru_size"`
+	LRUExpiration toml.Duration   `toml:"lru_expiration"`
```

Comment:
We don't need the `internal/toml/duration` this should work.
I tried it, and tests passed.
```suggestion
	UpdateFreq    time.Duration   `toml:"update_freq"`
	RateLimiter   RateLimitConfig `toml:"rate_limiter"`
	Redis         RedisConfig     `toml:"redis"`
	DefaultUsage  *int64          `toml:"default_usage"`
	LRUSize       int             `toml:"lru_size"`
	LRUExpiration time.Duration   `toml:"lru_expiration"`
```